.github/ci/.pre-commit-config.yaml

```diff
@@ -0,0 +1,30 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+exclude: ^examples/
```

Comment:
Why exclude `examples/`?

Reply:
Running this on examples folder was failing, so I removed it until I have time to decide wether code under examples should be also run under the same CI as tfx_addons package.

I initially also scoped only for CI to work for tfx-addons. I'm open to

Reply:
imo we should encourage good style + testing in examples. I'll take a look at why the tests were failing.

Reply:
I see, the current CI assumes the root dir is at the project root while the tests in the scikit-learn example assume the root dir the scikit-learn example project dir.

Also, the checks from `pre-commit-checks` are highly opinionated, and I'm not sure they align with our internal style. Not sure if we plan to migrate any more code from the TFX codebase, but that might make it difficult.

Reply:
We can disable the license addition part and isort, Yapf and pylint are mirroring TFX repo CI. 

Would love to discuss and find a good middleground for this. I added some that were useful but agree that we shouldnt make CI too hard either